app.js

```diff
@@ -14,12 +14,33 @@ app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
 
-// ...
+hbs.registerPartials(path.join(__dirname, 'views/partials'));
 
 // Add the route handlers here:
 
 app.get('/', (req, res) => {
   res.render('index');
 });
 
+
+app.get('/beers', (req, res) => {
+  punkAPI
+    .getBeers()
+    .then(beersFromApi => res.render('beers', {AllB: beersFromApi}))
+    .catch(error => console.log(error));
+});
+
+
+
+app.get('/random-beer', (req, res) => {
+  punkAPI
+    .getRandom()
+    .then(responseFromApi => res.render('random-beer', {RandomB: responseFromApi}))
```

Comment:
`responseFromApi` here is an array of one element. You could either pass the array as you did or you can also pass the beer object like so `{RandomB: responseFromApi[0]}`